OptimizelySDK/Utils/EventTagUtils.cs

```diff
@@ -16,5 +17,16 @@ public static object GetRevenueValue(Dictionary<string, object> eventTags)
 
            return eventTags[REVENUE_EVENT_METRIC_NAME];
         }
+
+        public static object GetEventValue(Dictionary<string, object> eventTags)
```

Comment:
We have to cover this code coverage as well.

Reply:
We already created a new unit-test named TestConversionEventWithNumericTag() for that.